src/app/common/utils/system-notifications/system-notifications.enums.ts

```diff
@@ -0,0 +1,11 @@
+export enum NotificationType {
+    Error = 'error',
+    Warning = 'warning',
+    Success = 'success',
+}
+
+export interface IShowNotificationConfig {
```

Comment:
Interfaces we should place in files ending in `.types.ts` - we should not keep them in the same file wit constants.